packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1042,7 +1042,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
           aria-selected={ isSelected ? 'true' : 'false'}
           ariaLabel= {item.text }
           disabled={ item.disabled }
-        > { <span ref={ this._selectedElement }>
+        > { <span ref={ isSelected ? this._selectedElement : '' }>
```

Comment:
Since we're trying to move away from string refs perhaps pass a `noop`? 

```ts
const noop = () => {};

...

> { <span ref={ isSelected ? this._selectedElement : noop }>
```

Reply:
What about undefined? TSLint won't pass with an empty function block

Reply:
Yeah, I think undefined would be fine, you may need to update the declaration to allow for undefined

Reply:
I think it's fine as is, ref is already an optional prop and passing undefined should just make it not set to anything. I'll push that in a minute